main.go

```diff
@@ -32,14 +32,16 @@ func main() {
 	// If port not available, raise FATAL error
 	serverFD, err := server.FindPortAndBind()
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
Do we need to change the behaviour of exiting from program if port is unavailable?

Reply:
Good call. I'll read up a bit more on the difference in behaviours. his was a change in logging libs and the Error should be the equivalent of Fatal from older lib. I'll confirm.

Reply:
I stand corrected. This was an unnecessary change. I will revert this.